modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -387,37 +387,78 @@ For example, you cannot have one xref:indexes-vector-similarity-euclidean[Euclid
 == Known issues
 
 The vector search index is a beta feature.
-The below table lists the known issues in its current implementation. 
+The below table lists the known issues in its current implementation.
 
-[%header,cols="5d,d"]
+[%header,cols="5a,d"]
 |===
 | Known issues | Fixed in
 
-a| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
+| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
 
 * {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`]
 * {link-procedures-reference}#procedure_db_index_vector_createNodeIndex[`db.index.vector.createNodeIndex`]
 * {link-procedures-reference}#procedure_db_index_vector_queryNodes[`db.index.vector.queryNodes`]
+
+[NOTE]
+====
+The procedures are still usable, just not visible.
+====
 | Neo4j 5.12
 
 | Passing `null` as an argument to some of the procedure parameters can generate a confusing exception.
-a| Neo4j 5.12
+| Neo4j 5.12
+
+| The creation of the vector index skipped the check to limit the dimensionality to `2048`.
 
-| The validation for xref:indexes-vector-similarity-cosine[cosine similarity] verifies the vector's {l2-norm} can be represented finitely in {ieee-754} *double* precision, rather than _single_ precision.
+[NOTE]
+====
+Vector indexes configured with dimensionality greater than `2048` in Neo4j 5.11 should continue to work after the limitation is applied.
+====
+|
+
+d| The validation for xref:indexes-vector-similarity-cosine[cosine similarity] verifies the vector's {l2-norm} can be represented finitely in {ieee-754} *double* precision, rather than _single_ precision.
```

Comment:
```suggestion
d| The validation for xref:indexes-vector-similarity-cosine[cosine similarity] verifies that the vector's {l2-norm} can be represented finitely in {ieee-754} *double* precision, rather than in _single_ precision.
```